core/src/main/java/org/apache/calcite/sql/SqlInsert.java

```diff
@@ -62,7 +62,11 @@ public SqlOperator getOperator() {
   }
 
   public List<SqlNode> getOperandList() {
-    return ImmutableNullableList.of(keywords, targetTable, source, columnList);
+    if (columnList == null) {
```

Comment:
Is it possible that `targetTable` is `null` or `source` is `null`?

Reply:
@DonnyZone Thansk for review. targetTable and source should be not null.
For example a sql such as `insert into emps` , it can’t be parsed.
`insert into emps VALUES (true, 1)` be parsed as SqlInsert, source not null.

Reply:
Thanks for clarification.